wolfssl/wolfcrypt/sha512.h

```diff
@@ -141,6 +141,10 @@ typedef struct wc_Sha512 {
 
 #endif /* HAVE_FIPS */
 
+#if defined(WOLFSSL_ARMASM) && defined(__aarch64__)
+int Transform_Sha512_Len(wc_Sha512* sha512, const byte* data, word32 len);
```

Comment:
For compilers obeying visibility shouldn't this be WOLFSSL_LOCAL?

Reply:
Fixed